test/E2ETest/testassets/projects/dockerfile/tye.yaml

```diff
@@ -0,0 +1,11 @@
+# tye application configuration file
+# read all about it at https://github.com/dotnet/tye
+name: frontend-backend
+services:
+- name: backend
+  dockerFile: backend/Dockerfile
+  bindings:
+    - port: 8002
```

Comment:
Random port

Reply:
You should be able to set containerPort to 80 (but I think it'll default for http) and set nothing else

Reply:
That needs to be fixed. Right now, a random port isn't specified (and I don't think one is specified for any non-project, but need to confirm that).

Reply:
Not sure what you mean. If you set containerPort a random port will get generated